doc/rst/source/gmt.conf.rst

```diff
@@ -422,6 +422,10 @@ GMT Miscellaneous Parameters
         or ~/.gmt. **Note**: Some of these languages may require you to also
         change the :term:`PS_CHAR_ENCODING` setting.
 
+    **GMT_MAX_CORES**
+        Sets the upper limit on the number of cores a multi-threaded module might
+        use if **-x** is selected [0, i.e., as many as possible].
```

Comment:
**GMT_MAX_CORES** only works if **-x** is selected?

How many cores are used for the following command?
```
gmt set GMT_MAX_CORES 2
gmt grdimage ...
```

Reply:
That is 1 unless you use -x and now you should get 2 (since you set that as a limit).

Reply:
Sorry, I'm more confused. Let's assume I have a 4-core PC:

- **grdimage**: 4 cores
- **grdimage -x**: 4 cores
- **grdimage -x1**: single core

After setting GMT_MAX_CORES=1, how many cores are used in the following commands?

- **grdimage**: 4 cores or single core?
- **grdimage -x**: single core
- **grdimage -x2**: 2 cores or single core?

Reply:
Initially, your available cores is 4. GMT_MAX_CORES (-0) is therefore 4.  However, if you set GMT_MAX_CORES to 1 then you effectively cannot use more that 1 core per process.  So grdimage -x2 will get 1 core.  It has to be this way so that your grdimage call in a movie script does not do -x4 and then we have 4 of those jobs running at the same time all running grdimage -x4.

Reply:
So with GMT_MAX_CORES=1 you only have one core no matter what -x you do.

Reply:
Then the description is misleading. When I read it, my understanding is that GMT_MAX_CORES has no effects if I don't use **-x**, thus **grdimage** without **-x** still uses 4 cores.
```
Sets the upper limit on the number of cores a multi-threaded module might
use if **-x** is selected [0, i.e., as many as possible].
```

Reply:
OK, will clarify.